src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -116,7 +118,7 @@
     private int retentionTimeout = DEFAULT_RETENTION_TIMEOUT_MINUTES;
     private int connectTimeout;
     private int readTimeout;
-    private Map<String, String> labels;
```

Comment:
To achieve backward compatibility, you need to keep the old field as `@deprecated / transient`, then migrate the data in `readResolve()` method.